<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Calibrate</class>
 <widget class="QWidget" name="Calibrate" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>UniMotion Calibrater</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_10" >
     <property name="text" >
      <string>Current</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_11" >
       <property name="text" >
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLineEdit" name="leCurX" />
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="label_12" >
       <property name="text" >
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLineEdit" name="leCurZ" />
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="label_13" >
       <property name="text" >
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="leCurY" />
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Maximum</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLineEdit" name="leMaxX" />
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLineEdit" name="leMaxZ" />
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="label_9" >
       <property name="text" >
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="leMaxY" />
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Model</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="leModel" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Minimum</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLineEdit" name="leMinX" />
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLineEdit" name="leMinZ" />
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="leMinY" />
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
